doc/releases/changelog-dev.md

```diff
@@ -9,6 +9,9 @@
 
 <h3>Improvements 🛠</h3>
 
+* Passing `shots` as a keyword argument to a `QNode` initialization now raises an error, instead of ignoring the input.
```

Comment:
Is it necessary/common practice to repeat the entry in 2 different changelog sections? This certainly seems to be a breaking change, but I wonder if it is worth mentioning this in the `Improvements` section. I usually assume that every breaking change leads to an improvement somehow (otherwise there would be no point in doing it)

Reply:
That's a good point :thinking: No strong preference from my side here. Could remove it here.